qml/HistoricStatisticsChartView.qml

```diff
@@ -82,4 +82,8 @@ StatisticsChartView {
         dateTimeAxisXMin = chartView.fromMsecsSinceEpoch(historicData.axisXMin(chartboxId))
         dateTimeAxisXMax = chartView.fromMsecsSinceEpoch(historicData.axisXMax(chartboxId))
     }
+
+    function customRemoveSeries(seriesIndex){

```

Comment:
Add this method to `StatisticsChartBox` so Dynamic and Historic shares it. (check Dynamic implementation as method already exist there).

Reply:
The implementation is not exactly the same. That's why there is one function for each chartview type